### PR TITLE
Mux stats late init

### PR DESCRIPTION
**Pull request by tomkordic**
https://github.com/muxinc/mux-stats-sdk-java/issues/24 fixed.

### CODE REVIEW

File: MuxExoPlayer/src/r2_10_6/java/com/mux/stats/sdk/muxstats/MuxStatsExoPlayer.java [hunk 1]
```diff
@@ -34,6 +37,12 @@ public MuxStatsExoPlayer(Context ctx, ExoPlayer player, String playerName, Custo
 
         if (player instanceof SimpleExoPlayer) {
             ((SimpleExoPlayer) player).addAnalyticsListener(this);
+            if (player.isPlaying()) {
+                // playback started before mux got intialized
+                play();
+                buffering();
+                playing();
+            }
```
Comment (jsanford8) [inline MuxExoPlayer/src/r2_10_6/java/com/mux/stats/sdk/muxstats/MuxStatsExoPlayer.java]: Is there a reason that in 2.10.x we do not need to check the other state? In the other versions, we check for two states (one that is already playing and one that is not), but not here. Just want to make sure I understand the change here.

**Review by jsanford8 — COMMENTED** [review]
Looks good to me, just have one question to understand the differences with 2.10.x vs the rest then we'll be good.

File: MuxExoPlayer/src/r2_10_6/java/com/mux/stats/sdk/muxstats/MuxStatsExoPlayer.java (hunk 1 above)
Comment (tomkordic) [reply MuxExoPlayer/src/r2_10_6/java/com/mux/stats/sdk/muxstats/MuxStatsExoPlayer.java]: Reason for this is that ExoPlayer change the API in 2.10 in such way that it no longer provide the player state so we only know if player is playing or not, in earlier versions we did not had player.isPlaying() method but instead we used playbackstate variable and compared it to existing player states.

**Comment by jsanford8** [comment]
Closing, pulled into v1.1.0

**Pull request closed without merging**